warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -115,6 +115,7 @@
 import org.apache.commons.lang.StringUtils;
 import org.apache.commons.lang.time.DateUtils;
 import org.apache.log4j.Logger;
+import org.graalvm.compiler.core.common.alloc.Trace;
```

Comment:
Am I missing where this is getting used?

Reply:
You're not, it was accidentally imported. I'll remove it.